apps/wallet/src/background/index.ts

```diff
@@ -88,8 +88,15 @@ accountsEvents.on('accountsChanged', async () => {
 		),
 	);
 });
-accountsEvents.on('accountStatusChanged', () => {
+accountsEvents.on('accountStatusChanged', async () => {
 	connections.notifyUI({ event: 'storedEntitiesUpdated', type: 'accounts' });
+	connections.notifyContentScript({
+		event: 'walletStatusChange',
+		origin,
+		change: {
+			accounts: await getAccountsStatusData(),
```

Comment:
I think this will return all the accounts of the wallet to the dapp. Not only the connected ones